packages/components/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -216,4 +216,175 @@ export const WithDecorator: Story = {
         </Tabs.Root>
     ),
 };
+
```

Comment:
the storybook controls are not working as the hardcoded padding always overrides the args.

if you switch the order it should work

Reply:
fixed